### DISCUSSION TITLE
No carga las facturas

### DISCUSSION THREAD

**Discussion by matalavera** [burkybang/SuiteAdvanced-Feedback#202] [General]
Hola, me percate que al instale la extension y buscar una factura y al abrirla no me la abre se queda dando vueltas el cursor y no carga la factura, realice una prueba desinstalando la extension y funciona normalmente, podrían revisarlo.

Saludos.

**Comment by burkybang** [comment]
I don't have this issue. Is your NetSuite instance in Spanish? Maybe there's a language incompatibility.
